src/readme.md

```diff
@@ -0,0 +1,30 @@
+# Rick and morty API
```

Comment:
By convention, the headers should have an empty line before the following paragraph
Please, do it for all the headers here